_studio/shared/umc/codec/jpeg_common/include/bitstreamin.h

```diff
@@ -58,7 +58,7 @@ class CBitStreamInput
 protected:
   CBaseStreamInput* m_in;
 
-  uint8_t*            m_pData;
+  uint8_t*          m_pData;
```

Comment:
I would prefer to see this change as a first commit in the series, not the last one.